docs/didkit/did-web.md

```diff
@@ -0,0 +1,153 @@
+we've created this page to walk you through the steps.
+
+## Picking a DID
+
+Did-web identifiers can denote a complete web domain or subdomain
+(`spruceid.com` or `demo.spruceid.com`), OR a subdirectory of the web-space at a
+domain (`demo.spruceid.com/path/to`). In either case, the represented site is
+referred to below as the `hosting site`. 
+
+*Note: as per the DID specification, pathnames must be entirely lowercase,
+regardless of the case-sensitivity of the web server in question. Similarly, in
+the case of identifiers that include a path, `:`s should be used instead of `/`s
+to conform to DID syntax, i.e. `https://demo.spruceid.com/path/to` would host
+the DID document for the DID `did:web:demo.spruceid.com:path:to`.*
+
+## Setup
+
+1. [Install DIDKit-CLI](/docs/didkit/install/) and any dependencies.
+   1. We recommend installing didkit-cli on the same server or virtual machine
+      that will run the issuance service and/or website that will issue
+      verifiable credentials signed by your `did:web`.  This allows you to
+      generate a JWK in situ in step 2 that will later by accessed locally by
+      the instance of DIDKit that will sign VCs.  
+
+2. Generate a fresh Ed25519 key and save it locally as a JWK with the command:
+
+```sh
+didkit generate-ed25519-key > issuer_key.jwk
+```
+
+*Note: We recommend using a fresh and unique private key generated on the web
+server and never moved elsewhere. Copying a private key from elsewhere is
+discouraged.*
+
+3. Generate a `did:key` from that Ed25519 key, display it, and save its DID
+   document locally:
+```sh
+did=$(didkit key-to-did key -k issuer_key.jwk)
+printf 'DID: %s\n\n' "$did"
+didkit did-resolve `didkit key-to-did key -k issuer_key.jwk` > issuer_key_did_doc.json
+```
+
+You should see a DID that looks like this
+ `did:key:z6MkwJBFYK8vTVGeiMsLzcqbSRXW4aTg4PozGbekWtQNUnnW` and the contents of
+ the file `issuer_key_did_doc.json` should look like this:
+```json
+{
+  "@context": "https://www.w3.org/ns/did/v1",
+  "id": "did:key:z6MkwJBFYK8vTVGeiMsLzcqbSRXW4aTg4PozGbekWtQNUnnW",
+  "verificationMethod": [
+    {
+      "id": "did:key:z6MkwJBFYK8vTVGeiMsLzcqbSRXW4aTg4PozGbekWtQNUnnW#z6MkwJBFYK8vTVGeiMsLzcqbSRXW4aTg4PozGbekWtQNUnnW", 
+      "type": "Ed25519VerificationKey2018",
+      "controller": "did:key:z6MkwJBFYK8vTVGeiMsLzcqbSRXW4aTg4PozGbekWtQNUnnW",
+      "publicKeyJwk": {
+        "kty": "OKP",
+        "crv": "Ed25519",
+        "x": "-kMHp5nohaFOK5E9Jch4ErdgwMFYFUc4Lt_wYlAGy8s"
+      }
+    }
+  ],
+  "authentication": [
+    "did:key:z6MkwJBFYK8vTVGeiMsLzcqbSRXW4aTg4PozGbekWtQNUnnW#z6MkwJBFYK8vTVGeiMsLzcqbSRXW4aTg4PozGbekWtQNUnnW"
+  ],
+  "assertionMethod": [
+    "did:key:z6MkwJBFYK8vTVGeiMsLzcqbSRXW4aTg4PozGbekWtQNUnnW#z6MkwJBFYK8vTVGeiMsLzcqbSRXW4aTg4PozGbekWtQNUnnW"
+  ]
+}
+```
+
+4. In the text editor of your choice, open the DID document file above. 
+    1. You'll want to change every instance of `did:key:z6Mkw...` to
```

Comment:
Here's a command to avoid doing it manually:
```bash
sed -i -e "s/did:key:z6MkwJBFYK8vTVGeiMsLzcqbSRXW4aTg4PozGbekWtQNUnnW/did:web:<my_website>/g" <my_file>.json
```